### ISSUE TITLE
sbteclipse-plugin 3.0.0: not found

### ISSUE THREAD

**Issue by klusht** [typesafehub/sbteclipse#251]
Hi , 

I am a beginner for Scala world, so I hope I don't miss to mention anything for this issue.

I have eclipse with scala plugin 4.0.0, and scala 2.11.5. 
Created a scala project, and added the plugin.sbt with 
addSbtPlugin("com.typesafe.sbteclipse" % "sbteclipse-plugin" % "3.0.0") , and I get the usual issue plugin not found

I have added additional resolver 
resolvers ++= Seq(
	"Sbteclipse"  at "http://repo.typesafe.com/typesafe/ivy-releases/"
) , but with no luck 

The plugin is available at
http://repo.typesafe.com/typesafe/ivy-releases/com.typesafe.sbteclipse/sbteclipse-plugin/scala_2.10/sbt_0.13/ .

Can someone support me how can I install this manually ? or the standard fix ? thanks

**Comment by khatkar** [comment]
@klusht  It should work without adding any resolvers. Can you provide contents of build.sbt and sbt logs that show this error?

**Comment by klusht** [comment]
Hi @khatkar I currently switched to intellij , and unfortunately I am not able to reproduce the issue. I will try to close this thread. Thanks for your support.

**Comment by cnokello** [comment]
I'm facing the same issue. The environment configs are the same as those of @klusht except I'm using Scala version 2.11.6. I don't have the IntelliJ option; I've to use Eclipse. Any thoughts on how I can go about sorting it out will be appreciated.

**Comment by cnokello** [comment]
I think that I might have found a solution. First, the default directory checked for plugins configuration is 'USER_HOME/.sbt/plugins' and NOT 'USER_HOME/.sbt/0.13/plugins'. Secondly, the sbt version specified seems to matter. In 'PROJECT_HOME/project/build.properties', there's an 'sbt.version' property. If the version of sbt specified in this file is different from the actually installed version of sbt, there's likely to be an issue. I think I was affected more so by this because I'm using an Activator template and it already sbt.version specified in this 'build.properties' file.

While nosing around, I came across this 

"Support for plugin configuration in project/plugins/ has been removed. It was deprecated since 0.11.2" from sbt website at http://www.scala-sbt.org/0.13.0/docs/Community/ChangeSummary_0.13.0.html.  I still see a lot of guys pointing that it should be configured in 'PROJECT_HOME/projects/plugins.sbt'. I think this is very confusing.
